nemo/collections/tts/data/text_to_speech_dataset.py

```diff
@@ -195,8 +195,8 @@ def _preprocess_manifest(
             sample = DatasetSample(
                 dataset_name=dataset_name,
                 manifest_entry=entry,
-                audio_dir=Path(dataset.audio_dir),
-                feature_dir=Path(dataset.feature_dir) if dataset.feature_dir is not None else None,
+                audio_dir=dataset.audio_dir,
```

Comment:
per `dataclass` definition, `audio_dir` and `feature_dir` are already the type of `Path`, so no need to enforce the conversion again.